### PR TITLE
Update keyword to work with V1 and V2 operator

### PR DESCRIPTION
**Pull request by tarukumar**

### CODE REVIEW

**Comment by tarukumar** [comment]
Smoke job #3199

**Review by lugi0 — APPROVED** [review]

**Review by aloganat — DISMISSED** [review]
lgtm

**Review by mwaykole — DISMISSED** [review]

**Review by lugi0 — APPROVED** [review]

File: ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot [hunk 1]
```diff
@@ -208,8 +208,13 @@ Is Component Enabled
     ${return_code}    ${output} =    Run And Return Rc And Output    oc get datasciencecluster ${dsc_name} -o json | jq '.spec.components.${component}\[]'  #robocop:disable
     Log    ${output}
     Should Be Equal As Integers	 ${return_code}	 0  msg=Error detected while getting component status
-    IF    ${output} == "Managed"
-        RETURN    true
-    ELSE IF    ${output} == "Removed"
-        RETURN    false
-    END
+    ${n_output} =    Evaluate    '${output}' == ''
```
Comment (bdattoma) [inline ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot:211]: doesn't ${EMPTY} work?

File: ods_ci/tasks/Resources/RHODS_OLM/uninstall/uninstall.robot [hunk 2]
```diff
@@ -70,10 +70,10 @@ Uninstall RHODS V2
     ...    oc delete dscinitialization $(oc get dscinitialization --no-headers | awk '{print $1}')
     Should Be Equal As Integers	${return_code}	 0   msg=Error deleting DSCInitialization CR
     ${return_code}    ${output}    Run And Return Rc And Output
-    ...    oc delete subscription $(oc get subscription -n redhat-ods-operator --no-headers | awk '{print $1}')
+    ...    oc delete subscription $(oc get subscription -n redhat-ods-operator --no-headers | awk '{print $1}') -n redhat-ods-operator
```
Comment (bdattoma) [inline ods_ci/tasks/Resources/RHODS_OLM/uninstall/uninstall.robot:73]: we should start using the global variable containing the namespace  - added by Jorge recently

**Review by bdattoma — APPROVED** [review]

File: ods_ci/tasks/Resources/RHODS_OLM/uninstall/uninstall.robot (hunk 2 above)
Comment (tarukumar) [reply ods_ci/tasks/Resources/RHODS_OLM/uninstall/uninstall.robot:73]: sure ..we can do that as well

**Pull request merged**